mon.pl

```diff
@@ -127,10 +127,19 @@
 
 warn "Logging in to $CONFIG->{homeserver} as $CONFIG->{user_id}...\n";
 
-$matrix->login(
-   user_id  => $CONFIG->{user_id},
-   password => $CONFIG->{password},
-)->get;
+my %login_details;
+if ($CONFIG->{password}) {
```

Comment:
You probably want to test with `defined` or `exists`, rather than mere truth; just in case of the offchance of the password "0" or some other silly cornercases.

Reply:
Thanks, changed!